api/rest/triple.yaml

```diff
@@ -0,0 +1,109 @@
+      - in: query
+        name: key
+        type: string
+        required: false
+        description: API key.
+      produces:
+      - application/json
+      responses:
+        '200':
+          description: A string encoding of a JSON object keyed by the given DCIDs with the value a list of associated triples.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  payload:
+                    type: string
+                    example: '{"dc/bsmvthtq89217":[{"subjectId":"dc/bsmvthtq89217","predicate":"country","objectId":"country/AUS","objectName":"Australia","objectTypes":["Country"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"continent","objectId":"oceania","objectName":"Oceania","objectTypes":["Continent"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"typeOf","objectId":"BiologicalSpecimen","objectName":"BiologicalSpecimen","objectTypes":["Class"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"provenance","objectId":"dc/93qydx3","objectName":"http://nybg.org/","objectTypes":["Provenance"],"provenanceId":"dc/93qydx3"}]}'
+        '400':
+          description: Missing argument 'dcids'.    
+    post:
+      summary: Given a list of nodes, return triples which are associated with the
+        specified node(s).
+      description: Intended for users who would like to modify only the bodies of
+        their cURL commands.
+      security:
+        - api_key: []
+      requestBody:
+        required:
+          - dcids
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/PostBody'
+            example:
+              dcids: ["dc/bsmvthtq89217"]
+              limit: 1
+      produces:
+      - application/json
+      responses:
+        '200':
+          description: A string encoding of a JSON object keyed by the given DCIDs with the value a list of associated triples.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  payload:
+                    type: string
+                    example: '{"dc/bsmvthtq89217":[{"subjectId":"dc/bsmvthtq89217","predicate":"country","objectId":"country/AUS","objectName":"Australia","objectTypes":["Country"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"continent","objectId":"oceania","objectName":"Oceania","objectTypes":["Continent"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"typeOf","objectId":"BiologicalSpecimen","objectName":"BiologicalSpecimen","objectTypes":["Class"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"provenance","objectId":"dc/93qydx3","objectName":"http://nybg.org/","objectTypes":["Provenance"],"provenanceId":"dc/93qydx3"}]}'
```

Comment:
Can we format this better? Not sure if it's possible to add line breaks in here. Technically, the returned response is full of escape characters.

I wonder if we can do something similar to what we have, where we just say you get a string blob, but here's what it is if you json.parse it again.